File: Python projects/Python Basics/advanced_conditional_statements/lab/personal_titles.py
age = float(input())
gender = str(input())

title = ""

if age >= 16:
    if gender == "m":
        title = "Mr."
    else:
        title = "Ms."
else:
    if gender == "m":
        title = "Master"
    else:
        title = "Miss"

print(title)

File: JavaScript projects/JavaScript Basics/Conditional Statements Advanced/lab/personal_titles.js
function personalTitles(age, gender){
    switch(gender){
        case "m":
            if (16 <= age){
                console.log("Mr.");
            }else{
                console.log("Master");
            }
            break; 
        case "f":
            if (16 <= age){
                console.log("Ms.");
            }else{
                console.log("Miss");
            } 
            break;
        }
}

personalTitles(12,"f");